pom.xml

```diff
@@ -225,13 +231,15 @@
               <encoding>UTF-8</encoding>
               <consoleOutput>true</consoleOutput>
               <failsOnError>true</failsOnError>
-              <linkXRef>false</linkXRef>
             </configuration>
             <goals>
               <goal>check</goal>
             </goals>
           </execution>
         </executions>
+        <configuration>
+          <linkXRef>false</linkXRef>
```

Comment:
I did not change anything in the configuration, this prop was already there:
https://github.com/spotify/docker-maven-plugin/blob/master/pom.xml#L228
I just moved it outside of the execution step as my IDE was complaining that is was against validate phase configuration schema.